launcher/minecraft/mod/ResourceFolderModel.cpp

```diff
@@ -500,6 +505,67 @@ QVariant ResourceFolderModel::headerData(int section, Qt::Orientation orientatio
     return {};
 }
 
+void ResourceFolderModel::setupHeaderAction(QAction* act, int column)
+{
+    Q_ASSERT(act);
+
+    act->setText(headerData(column, Qt::Orientation::Horizontal).toString());
+}
+
+void ResourceFolderModel::saveHiddenColumn(int column, bool hidden)
+{
+    auto const setting_name = QString("UI/%1_Page/HiddenColumns").arg(id());
```

Comment:
shouldn't this use the settings object of the instance associated with this model?

Reply:
Should the setting really be instance specific? I suppose it could be but I was thinking it would be better as a application level setting.